all_test.go

```diff
@@ -720,6 +733,34 @@ func TestMaxSessionLimit(t *testing.T) {
 	}
 }
 
+func TestSqlSessionCache(t *testing.T) {
+	c := setup1(t)
+	defer Teardown(c)
+	c.MaxActiveSessions = 1
+
+	// Login first time
+	key, _, _ := c.Login(joeIdentity, joePwd)
```

Comment:
Is this intended to test the capitalization of the login name? If so, we should say so in comments, and also login with a differently cased username to the actual "joeIdentity"